templates/apps/polymorph-prompt.hbs

```diff
@@ -1,22 +1,36 @@
-<div class="dialog-content">
-	{{#each content.i18n}}
-		<div>
-			<label class="checkbox" for="{{@key}}">
-				<input type="checkbox" id="{{@key}}" name="{{@key}}" {{checked (lookup ../content.options @key)}}>
-				{{this}}
-			</label>
-		</div>
-	{{/each}}
-	{{#unless content.isToken}}
-	<hr>
-	<div>
-		<label class="checkbox" for="transformTokens">
-			<input type="checkbox" id="transformTokens" name="transformTokens"
-				{{checked content.options.transformTokens}}>
-			{{localize 'DND5E.PolymorphTokens'}}
-		</label>
-	</div>
-	{{/unless}}
+<div class="polymorph two-column">
+    <div class="left-column">
+		<h3>{{localize 'DND5E.PolymorphCustomTransformation'}}</h3>
```

Comment:
Looks like some tabs snuck in. I know the original was tabs too, but it's good to correct it now.